config-scripts/cups-compiler.m4

```diff
@@ -114,8 +114,8 @@ AS_IF([test -n "$GCC"], [
 
     # Default optimization options...
     AS_IF([test -z "$OPTIM"], [
-	# Default to optimize-for-size and debug
-	OPTIM="-Os -g"
+	# Default to optimize-for-size and LTO
```

Comment:
LTO isn't generally available, so I'd rather keep with the current default and let distros/packagers tweak things for their particular versions of GCC/Clang/etc.